pkg/cmd/roachprod/main.go

```diff
@@ -118,14 +118,19 @@ func newCluster(name string, reserveLoadGen bool) (*install.SyncedCluster, error
 
 	c, ok := install.Clusters[name]
 	if !ok {
-		return nil, fmt.Errorf(`unknown cluster: %s
+		// NB: We don't use fmt.Errorf due to a linter error about the error
+		// message containing capitals and punctuation. We don't use
```

Comment:
I hate that linter warning about the capitalization. And does it really complain about punctuation? I feel like I put multiple sentences with periods between them in errors all the time.
Let's rip out the linter, now that Tamir is gone.

Reply:
Feel free to do that, though I thought this error was baked into `golint`. Perhaps I'm mistaken about that.